strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/main/java/org/carlspring/strongbox/artifact/coordinates/PypiWheelArtifactCoordinates.java

```diff
@@ -0,0 +1,238 @@
+    }
+
+    /**
+     * @param path: The filename of the PyPi Wheel package
+     * @return Returns a PyPiWheelArtifactCoordinates object with all included coordinates set
+     */
+    public static PypiWheelArtifactCoordinates parse(String path)
+    {
+        return PypiWheelArtifactCoordinatesUtils.parse(path);
+    }
+
+    /**
+     * @return Returns distribution coordinate value (serves as the unique ID)
+     */
+    @Override
+    public String getId()
+    {
+        return getCoordinate(DISTRIBUTION);
+    }
+
+    /**
+     * @param id: DISTRIBUTION coordinate will take this value
+     */
+    @Override
+    public void setId(String id)
+    {
+        setCoordinate(DISTRIBUTION, id);
+    }
+
+    /**
+     * @return Returns the VERSION coordinate value
+     */
+    @Override
+    public String getVersion()
+    {
+        return getCoordinate(VERSION);
+    }
+
+    /**
+     * @param version: VERSION coordinate takes this value
+     */
+    @Override
+    public void setVersion(String version)
+    {
+        setCoordinate(VERSION, version);
+    }
+
+    /**
+     * @return Returns the BUILD_TAG coordinate value
+     */
+    @ArtifactLayoutCoordinate
+    public String getBuild()
+    {
+        return getCoordinate(BUILD_TAG);
+    }
+
+    /**
+     * @param build: BUILD_TAG coordinate will take this value
+     */
+    public void setBuild(String build)
+    {
+        setCoordinate(BUILD_TAG, build);
+    }
+
+    /**
+     * @return Returns the LANGUAGE_IMPLEMENTATION_VERSION_TAG coordinate value
+     */
+    @ArtifactLayoutCoordinate
+    public String getLanguageImplementationVersion()
+    {
+        return getCoordinate(LANGUAGE_IMPLEMENTATION_VERSION_TAG);
+    }
+
+    /**
+     * @param lang: LANGUAGE_IMPLEMENTATION_VERSION_TAG takes this value
+     */
+    public void setLanguageImplementationVersion(String lang)
+    {
+        setCoordinate(LANGUAGE_IMPLEMENTATION_VERSION_TAG, lang);
+    }
+
+    /**
+     * @return Returns the ABI_TAG coordinate value
+     */
+    @ArtifactLayoutCoordinate
+    public String getAbi()
+    {
+        return getCoordinate(ABI_TAG);
+    }
+
+    /**
+     * @param abi: ABI_TAG coordinate takes this value
+     */
+    public void setAbi(String abi)
+    {
+        setCoordinate(ABI_TAG, abi);
+    }
+
+    /**
+     * @return Returns the PLATFORM_TAG coordinate value
+     */
+    @ArtifactLayoutCoordinate
+    public String getPlatform()
+    {
+        return getCoordinate(PLATFORM_TAG);
+    }
+
+    /**
+     * @param platform: PLATFORM_TAG coordinate takes this value
+     */
+    public void setPlatform(String platform)
+    {
+        setCoordinate(PLATFORM_TAG, platform);
+    }
+
+    /**
+     * @return Returns the reconstructed path from the stored coordinate values
+     */
+    @Override
+    public String toPath()
+    {
+        // if optional BUILD_TAG coordinate is empty, don't include it in the reconstruction
+        if (getBuild() == "")
```

Comment:
Please, use `StringUtils.isBlank()` instead.

Reply:
Should be resolved in all instances now.